src/common/location_summaries.ts

```diff
@@ -62,7 +62,7 @@ export function useSummaries(): SummariesMap | null {
       setSummaries(summaries);
     }
     fetch();
-  });
```

Comment:
This seems fine, but rather than factoring `snapshot` out and then making it a dependency, could you just add `[]` as the dependency list here?

Reply:
I could.  It would mean it wouldn't be updated if we ever make snapshot update outside of a navigation, e.g., something like a built-in that auto-cycles through data like a gif-maker without navigation.

But maybe WAGNI, so sure I can make that change.